compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -144,6 +144,9 @@ void buildGlobalOptimizationPassPipeline(
                 transformOptions.options.aggressiveTransposePropagation);
           })
       .addPass(mlir::createCanonicalizerPass)
+      .addPass(mlir::createCSEPass)
+      .addPass(createFuseHorizontalContractionsPass)
+      .addPass(mlir::createCanonicalizerPass)
       .addPass(mlir::createCSEPass);
```

Comment:
Do we want this on by default?

Reply:
My understanding is that this is good for GPU, but I'm not sure if it is applied to other backends. We should at least run some benchmarks?

Reply:
No, we dont want it on by default (at least for now)